recipes/ligo-remote-epics-scripts/meta.yaml

```
@@ -0,0 +1,35 @@
+{% set name = "ligo-remote-epics-scripts" %}
+{% set version = "0_0_3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace('_','.') }}
+
+source:
+  url: https://git.ligo.org/cds/ligo-remote-epics-scripts/-/archive/v0.0.3/ligo-remote-epics-scripts-v0.0.3.tar.gz
```

If the version in the repo is `x.y.z`, just use that (rather than underscores) and then use `{{ version }}` to avoid repeating the version number in the rest of the recipe:

```suggestion
{% set version = "0.0.3" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://git.ligo.org/cds/ligo-remote-epics-scripts/-/archive/v{{ version }}/ligo-remote-epics-scripts-v{{ version }}.tar.gz
```